src/main/java/uk/gov/hmcts/reform/slc/services/SendLetterService.java

```diff
@@ -46,7 +51,8 @@ public MessageHandlingResult send(IMessage msg) {
             letter = letterMapper.from(msg);
             PdfDoc pdf = pdfCreator.create(letter);
             // TODO: encrypt & sign
-            ftpClient.upload(pdf, isSmokeTest(letter));
+            ZippedDoc zippedDoc = zipper.zip(Zipper.generateName(letter), pdf);
```

Comment:
You could call the `generateName` from the `zip` method itself as it is available in the same class?

Reply:
it's a static method